test_security/CMakeLists.txt

```diff
@@ -274,7 +274,8 @@ if(BUILD_TESTING)
       (rmw_implementation STREQUAL "rmw_connext_cpp" AND NOT WIN32) OR
       (rmw_implementation STREQUAL "rmw_connext_dynamic_cpp" AND NOT WIN32) OR
       rmw_implementation STREQUAL "rmw_fastrtps_cpp" OR
-      rmw_implementation STREQUAL "rmw_fastrtps_dynamic_cpp"
+      rmw_implementation STREQUAL "rmw_fastrtps_dynamic_cpp" OR
```

Comment:
Should the TODO above be updated?

Reply:
Good point, I simply removed it in https://github.com/ros2/system_tests/pull/408/commits/8876c0a2da5b6da298143ca0e3ab8ecba5093856